.gitmodules

```diff
@@ -25,3 +22,9 @@
 [submodule "src/extensions/default/JSLint/thirdparty/jslint"]
 	path = src/extensions/default/JSLint/thirdparty/jslint
 	url = https://github.com/peterflynn/JSLint.git
+[submodule "src/extensions/default/JavaScriptCodeHints/node/node_modules/acorn"]
```

Comment:
Ooc, would not be better to create a `package.json` in `src/extensions/default/JavaScriptCodeHints/node`?